src/napari_workflow_inspector/_dock_widget.py

```diff
@@ -1,26 +1,57 @@
 import numpy as np
 from napari_plugin_engine import napari_hook_implementation
 from qtpy.QtWidgets import QWidget, QHBoxLayout, QVBoxLayout, QPushButton, QLayout, QLabel, QTabWidget, QScrollArea
+from qtpy.QtWidgets import QFileDialog
 from qtpy.QtWidgets import QSpacerItem, QSizePolicy
 from qtpy.QtCore import QTimer, Qt
 from napari_tools_menu import register_dock_widget
 
 from matplotlib.backends.backend_qt5agg import FigureCanvasQTAgg as FigureCanvas
+from matplotlib.backends.backend_qt5 import NavigationToolbar2QT as NavigationToolbar
 from matplotlib.figure import Figure
 import matplotlib
 matplotlib.use('Qt5Agg')
 
-# Adapted from https://github.com/BiAPoL/napari-clusters-plotter/blob/main/napari_clusters_plotter/_plotter.py
+import pickle
+import networkx as nx
+
+
+# Adapted from https://github.com/jo-mueller/RadiAiDD/blob/master/RadiAIDD/Backend/UI/_matplotlibwidgetFile.py
 class MplCanvas(FigureCanvas):
+    """
+    Defines the canvas of the matplotlib window
+    """
 
-    def __init__(self, parent=None, width=7, height=4):
-        self.fig = Figure(figsize=(width, height))
+    def __init__(self):
+        self.fig = Figure()                         # create figure
+        self.axes = self.fig.add_subplot(111)       # create subplot
+        self.fig.subplots_adjust(left=0.04, bottom=0.04, right=0.97,
+                                 top=0.96,  wspace=None, hspace=None)
 
-        # changing color of axis background to napari main window color
         self.fig.patch.set_facecolor('#262930')
-        self.axes = self.fig.add_subplot(111)
-
-        super(MplCanvas, self).__init__(self.fig)
+        self.axes.set_facecolor('#262930')
```

Comment:
I'm speculating that this line should be removed to have a transparent background,